clang/lib/CodeGen/CGBuiltin.cpp

```diff
@@ -66,6 +66,11 @@ using namespace clang;
 using namespace CodeGen;
 using namespace llvm;
 
+static llvm::cl::opt<bool> ClSanitizeAlignmentBuiltin(
+    "sanitize-alignment-builtin", llvm::cl::Hidden,
+    llvm::cl::desc("Instrument builtin functions for -fsanitize=alignment"),
+    llvm::cl::init(true));
+
```

Comment:
Why are you using a `cl::opt` here rather than a proper `CodeGenOption`? We generally don't use `cl::opt` in clang and want our options to be provided via our option structs instead, so that clang can be used as a library.

Reply:
I acknowledge that `cl::opt` is discouraged (`clang/lib` has very few of them). This is a temporary workaround (so that our users can perform cleanups) and should not be here for a too-long time. I'll revert this after ~two months, assuming that rolling update users (like us) have caught up on the change. Given how large our code base is and how few (relatively) problems identified, I reasonably expect that we don't really need a driver option or cc1 option.

I modeled this after `cl::opt<bool> ClSanitizeOnOptimizerEarlyEP` in `clang/lib/CodeGen/BackendUtil.cpp` and thought that cl::opt might look more experimental than a cc1 option...